src/plugins/discover/public/application/main/components/sidebar/discover_field_search.tsx

```diff
@@ -294,8 +365,58 @@ export function DiscoverFieldSearch({ onChange, value, types }: Props) {
             {selectionPanel}
             {footer()}
           </EuiPopover>
+          <EuiPopover
+            anchorPosition="rightUp"
+            display="block"
+            button={helpButton}
+            isOpen={isHelpOpen}
+            panelPaddingSize="m"
+            className="dscFieldTypesHelp__popover"
+            panelClassName="dscFieldTypesHelp__panel"
+            closePopover={closeHelp}
+          >
+            <EuiPopoverTitle paddingSize="s">
+              {i18n.translate('discover.fieldChooser.popoverTitle', {
+                defaultMessage: 'Field types',
+              })}
+            </EuiPopoverTitle>
+            <EuiBasicTable
+              tableCaption={i18n.translate('discover.fieldTypesPopover.tableTitle', {
+                defaultMessage: 'Description of field types',
+              })}
+              items={items}
+              compressed={true}
+              rowHeader="firstName"
+              columns={columnsSidebar}
+              responsive={false}
+            />
+            <EuiSpacer size="s" />
+            {presentFieldTypes.length > FIELD_TYPES_PER_PAGE && (
+              <EuiFlexGroup justifyContent="flexEnd" gutterSize="none" responsive={false}>
+                <EuiFlexItem grow={false}>
+                  <EuiPagination
```

Comment:
I **really** don't think pagination is necessary here, there's only like 10 fields that have descriptions right now and can often leave stragglers (1 row) on a single page. My suggestion is just to wrap the table in an overflowing container once it reaches a certain height. I'll setup a PR against this to showcase what I mean but essentially it would behave like this: 

https://user-images.githubusercontent.com/549577/161828833-3397c362-87f1-4202-8a54-5a7bb6bf172c.mp4

Reply:
PR4U: https://github.com/andreadelrio/kibana/pull/7